src/signed-api-fetch/data-fetcher.ts

```diff
@@ -89,7 +89,6 @@ export const runDataFetcher = async () => {
             localDataStore.setStoreDataPoint(element);
           }
         },
-        // TODO: What to do about timeout and retries? This is the only place that does some handling.
```

Comment:
https://github.com/api3dao/airseeker-v2/issues/91

As a note. This is the only axios call at the moment. The thing that should change is the timeout. It should not be a static value but instead depend on the `signedDataFetchInterval`.